test/cmdlineTests/standard_urls_existing_and_missing/output.json

```diff
@@ -0,0 +1 @@
+{"errors":[{"component":"general","formattedMessage":"Cannot import url (\"standard_urls_existing_and_missing/non-existent-contract-1.sol\"): File not found. Searched the following locations: \"\".","message":"Cannot import url (\"standard_urls_existing_and_missing/non-existent-contract-1.sol\"): File not found. Searched the following locations: \"\".","severity":"warning","type":"Warning"}],"sources":{"url_not_found.sol":{"id":0}}}
```

Comment:
Looks like the `args` files disappeared here and in the other JSON test so this reverted back to being unformattted.

Reply:
Yea. That's what made the tests pass. Else we were having the issues brought up in the comment here https://github.com/ethereum/solidity/pull/13162#discussion_r967667609

Reply:
But you can also make it pass by doing the small adjustment suggested there, right? This will only be a problem if this tests needs to be autoupdates which is not very often.

Reply:
No, there are issues with the adjustments either way.

Whichever way the adjustment goes, one of 
https://app.circleci.com/pipelines/github/ethereum/solidity/26515/workflows/1537a81d-d798-4598-a7e3-4743160ca2c3/jobs/1174412 
and
https://app.circleci.com/pipelines/github/ethereum/solidity/26478/workflows/326ca1f8-e91e-4034-ba21-dfac16cd2b86/jobs/1172659

fails. 

So, far, removing the `args` file seems to be the only quick solution. Unless of course we go in depth and examine why this behaviour is happening.

Reply:
For now, I'll proceed with removing the `args` file to enable the merge to go ahead.

Reply:
Ok, right. I thought it was just consistently producing misformatted output but it just produces different output between release and pre-release builds. In that case I know what it it is. Reported as: #13544.

Removing `--pretty-print` in this PR is fine then. We'll re-enable it in #7665.